defaults.sh

```diff
@@ -103,12 +103,12 @@ BUILD_TOOLS_SRC=$(cd "$(dirname "$0")" && pwd)
 # Qt
 
 # Source directory
-DEF_QT_SOURCE_PACKAGE=qt-everywhere-opensource-src-$DEF_QT_VER
+DEF_QT_SOURCE_PACKAGE=qt-everywhere-src-$DEF_QT_VER
 DEF_QT_SRC_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE
 
 if [[ $UNAME_SYSTEM == "Linux" ]] || [[ $UNAME_SYSTEM == "Darwin" ]]; then
     # Dynamic Qt build directory on Linux and MacOS
-    DEF_QT_DYN_BUILD_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE-build
+    DEF_QT_DYN_BUILD_DIR=$DEF_PREFIX/invariant/$DEF_QT_SOURCE_PACKAGE-build-rpathpadrpathpad
```

Comment:
rpathpadrpathpad?

Reply:
lovely, right? :)

Reply:
It sure is, but what's the reason behind that?

Reply:
RPATH is used to resolve library dependencies under user provided installation dir. RPATH is modified during Qt Creator's make-install phase when Qt is packaged. New RPATH needs to fit into the space occupied by the original RPATH. Hence the padding to reserve more space. The original name contained the word "-opensource" which did the trick. Similar issue arises every now and then.

rpath-pad - self documenting :)

Reply:
I was quite close with my guesses then. I was just a bit confused why we didn't need it before.